docs/docs/faq.md

```diff
@@ -80,6 +80,10 @@ Please note, many keyboards only have a single PCB which includes the “brains
 
 Currently, ZMK only supports wireless split, but wired split is possible and we welcome contributions!
 
+### How is the latency?
+
+The latency of ZMK is comparable to other firmware offerings. ZMK is equipped with a variety of scanning methods and debounce algorithms that can affect the final measured latency. [This video](https://www.youtube.com/watch?v=jWL4nU-vtWs) shows a latency comparison of ZMK and other keyboard firmwares.
```

Comment:
Can we link to the debounce docs as well from here?

I wonder if it might be worthwhile to also add an entry specifically talking about "2.4ghz dongles" and the plan/non-plan there.

Reply:
> I wonder if it might be worthwhile to also add an entry specifically talking about "2.4ghz dongles" and the plan/non-plan there.

@Hello9999901 has added an entry with commit a197bf8.

I think a section regarding "2.4ghz dongles" would be another good addition to the FAQ although I wonder if perhaps this too would be best discussed in separate PR, so we can merge this "how is the latency" section now?

Reply:
I was talking about this with Pete, and he said to have them together in the same PR.